client/js/libs/handlebars/parse.js

```diff
@@ -83,6 +91,8 @@ module.exports = function parse(text) {
 			return `<span class="inline-channel" role="button" tabindex="0" data-chan="${escapedChannel}">${fragments}</span>`;
 		} else if (textPart.emoji) {
 			return `<span class="emoji">${fragments}</span>`;
+		} else if (textPart.nick) {
+			return templates.user_name({nick: textPart.nick}).trim();
```

Comment:
Unfortunately to fix tests (at least right now), the easiest solution would be copying user_name template here.

Tests fail because .tpl files aren't being compiled when required from mocha.

Also probably should be using `fragments` here like the others above.

Reply:
Done